docs/contributing.rst

```diff
@@ -75,9 +75,9 @@ The NumCodecs source code is hosted on GitHub at the following location:
 You will need your own fork to work on the code. Go to the link above and hit
 the "Fork" button. Then clone your fork to your local machine::
 
-    $ git clone git@github.com:your-user-name/numcodecs.git
```

Comment:
This goal of this line (based on the proceeding text) is to checkout the user's own fork and not the main repo.

Reply:
Oh yes, missed that!

Reply:
Thanks. So now my only question is if someone doesn't have SSH locally, will this allow them to push a change?

Reply:
Yes! I personally don't use SSH and it works fine.

Reply:
Personally use `ssh` and know the `git@...` syntax is needed to push a change (`https://...` doesn't work)

Reply:
Updated!

Reply:
Thanks! Though AIUI this doesn't work for you. So maybe we should show both syntaxes? Tried a pass at that below